.github/workflows/ci.yml

```diff
@@ -155,8 +155,10 @@ jobs:
 
       - name: Assembly alias
         run: |
+          Get-ChildItem -Path "package-dev/Runtime/" -name
           dotnet tool install --global Alias --version 0.2.0
           assemblyalias --target-directory "package-dev/Runtime" --internalize --prefix "Sentry." --assemblies-to-alias "Microsoft*;System*"
+          Get-ChildItem -Path "package-dev/Runtime/" -name
```

Comment:
Out of curiosity, isn't line 161 doing the same thing as line 158?

Reply:
Yes! I just wanted to verify that line 160 actually does rename the files.